l10n_it_split_payment/models/account.py

```diff
@@ -112,25 +119,39 @@ def _compute_split_payments(self):
         line_sp = self.line_ids.filtered(lambda l: l.is_split_payment)
         if line_sp:
             if self.move_type == "out_invoice" and float_compare(
-                line_sp[0].debit,
-                write_off_line_vals["debit"],
+                line_sp.price_unit,
```

Comment:
Qui e più sotto rimetterei `line_sp[0]` così come già discusso in https://github.com/OCA/l10n-italy/pull/1959#discussion_r602127514 Cosa ne pensi?

Reply:
Vedi https://github.com/Borruso/l10n-italy/pull/11/commits/89bc3f3bdaa0f4e22954f8b8ec620497e4741f07